docs/topics/health-check-mission-basic-interaction-wf-swarm.adoc

```diff
@@ -8,19 +8,19 @@ In the following we demonstrate the basic interactions on the command line that
 +
 [source,bash,options="nowrap",subs="attributes"]
 ----
-$ curl http://{app-name}-{project-name}.192.168.64.3.nip.io/api/greeting
+$ curl http://{app-name}-{project-name}.1ab5.starter-us-east-1.openshiftapps.com/api/greeting
 {"content":"Hello, World!"}
 ----
 
 . To simulate a service internal failure and to trigger the openshift selfhealing capabilities, invoke the `/api/killme` endpoint and verify the availability of the `/api/greeting` endpoint shortly after that. It should return a HTTP status 503:
```

Comment:
I think OpenShift should be written in camel case. WDYT?

openshift > OpenShift